.gitignore

```diff
@@ -14,3 +14,4 @@ draft-ietf-sedate-datetime-extended.xml
 lib
 report.xml
 venv/
+.vscode
```

Comment:
Not to be a pedant, but shouldn't this be in your global .gitignore instead? :sweat_smile:

Reply:
Hmmm, that works too... it's just my habit in other open source projects to make things easier for other VSCode users. Also fine to ignore this change. Your call!

Reply:
My usual rule of thumb is: if it's specific to a given project: include it in the project, if it's specific to your work setup: include it in the global gitignore. This would keep you from repeating yourself :innocent:

Reply:
```suggestion
```